src/test/java/com/minecolonies/coremod/colony/ColonyTest.java

```diff
@@ -91,33 +80,36 @@ public void setup() throws NoSuchFieldException, IllegalAccessException
     public void testToggleAnything()
     {
         colony.setName("blahsColony");
+        colony.writeToNBT(new NBTTagCompound());
```

Comment:
Even without this call, it shouldn't be null, because you check in getColonyTag(). So you don't need this line.